plugins/console/index.js

```diff
@@ -0,0 +1,127 @@
+var q = require('q');
+
+var testOut = {failedCount: 0, specResults: []};
```

Comment:
You haven't quite conformed to [the spec](https://github.com/angular/protractor/tree/master/lib/frameworks) here. I think what you want is:

```js
var testOut = {failedCount: 0, specResults = [{
   description: "Console output",
   assertions: [],
   duration: 0
}]};
```

And then, in later lines, when you write `testOut.specResults.push(` you want `testOut.specResults.assertions.push(`